ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -353,39 +359,54 @@ public Task<bool> IsPackagerRunningAsync()
             HideRedboxDialog();
             HideDevOptionsDialog();
 
-            var message = !IsRemoteDebuggingEnabled
+            Action cancel;
+            CancellationToken token;
+
+            if (IsProgressDialogEnabled)
+            {
+                var message = !IsRemoteDebuggingEnabled
                 ? "Fetching JavaScript bundle."
                 : "Connecting to remote debugger.";
 
-            var progressDialog = new ProgressDialog("Please wait...", message);
+                ProgressDialog progressDialog = new ProgressDialog("Please wait...", message);
+
 #if WINDOWS_UWP
-            var dialogOperation = progressDialog.ShowAsync();
-            Action cancel = dialogOperation.Cancel;
+                var dialogOperation = progressDialog.ShowAsync();
+                cancel = dialogOperation.Cancel;
 #else
-            if (Application.Current != null && Application.Current.MainWindow != null && Application.Current.MainWindow.IsLoaded)
-            {
-                progressDialog.Owner = Application.Current.MainWindow;
+                if (Application.Current != null && Application.Current.MainWindow != null && Application.Current.MainWindow.IsLoaded)
+                {
+                    progressDialog.Owner = Application.Current.MainWindow;
+                }
+                else
+                {
+                    progressDialog.Topmost = true;
+                    progressDialog.WindowStartupLocation = WindowStartupLocation.CenterScreen;
+                }
+
+                cancel = progressDialog.Close;
+                progressDialog.Show();
+#endif
+                token = progressDialog.Token;
             }
             else
             {
-                progressDialog.Topmost = true;
-                progressDialog.WindowStartupLocation = WindowStartupLocation.CenterScreen;
+                // Progress not enabled - provide empty implementations
+                cancel = () => { };
+                token = default(CancellationToken);
```

Comment:
>default(CancellationToken); [](start = 24, length = 27)

Not sure if there's any difference between default(CancellationToken) and CancellationToken.None. Presumably not.